pkg/controller/operators/olm/initializer/config.go

```diff
@@ -16,7 +16,7 @@ type operatorConfig struct {
 	logger *logrus.Logger
 }
 
-func (o *operatorConfig) GetOperatorConfig(ownerCSV ownerutil.Owner) (overrides []corev1.EnvVar, err error) {
+func (o *operatorConfig) GetOperatorConfig(ownerCSV ownerutil.Owner) (overrides []corev1.EnvVar, volumes []corev1.Volume, volumeMounts []corev1.VolumeMount, err error) {
```

Comment:
since we know we need to expand the config in the future, we could return some struct `overrides *ConfigOverrides`

Reply:
That's a good idea, I'll also update the variables returned to `envVarOverrides`, `volumeOverrides`, and `volumeMountOverrides`.